Makefile.in

```diff
@@ -109,12 +110,18 @@ user-install:
 	$(info :: Installed under user-only location "$(DATADIR)/$(INSTALLDIR)")
 	$(info :: To run you need to make sure "$(BINDIR)" is in your PATH)
 
+.PHONY: docs
+ulwgl-docs-install:
+	$(info :: Installing man pages)
+	@mkdir -p $(DESTDIR)$(MANDIR)/man1
+	scdoc < docs/ulwgl.1.scd > $(OBJDIR)/ulwgl.1
+	install -m644 $(OBJDIR)/ulwgl.1 $(DESTDIR)$(MANDIR)/man1/ulwgl.1
```

Comment:
This preferably should be two steps, one to build the manpage and a second step to install the manpage.